staging/src/k8s.io/apimachinery/pkg/api/validation/objectmeta.go

```diff
@@ -53,8 +53,8 @@ func ValidateAnnotations(annotations map[string]string, fldPath *field.Path) fie
 		}
 		totalSize += (int64)(len(k)) + (int64)(len(v))
 	}
-	if totalSize > (int64)(totalAnnotationSizeLimitB) {
-		allErrs = append(allErrs, field.TooLong(fldPath, "", totalAnnotationSizeLimitB))
+	if totalSize > (int64)(TotalAnnotationSizeLimitB) {
+		allErrs = append(allErrs, field.TooLong(fldPath, "", TotalAnnotationSizeLimitB))
```

Comment:
Both the constant and the mechanism used to sum the key/value sizes should be exposed, so we don't have to duplicate the addition logic

e.g.

```go
func ValidateAnnotationsSize(annotations map[string]string, fldPath *field.Path) error {
  ...
}
```

Reply:
Done